.eslintrc

```diff
@@ -0,0 +1,44 @@
+{
+  "env": {
+    "node": true
+  },
+  "globals": {},
+  "rules": {
+    "no-bitwise": 0,
+    "camelcase": 0,
```

Comment:
Hehe.. at work we actually are explicit about all the rules we configure - have a look at https://github.com/holidaycheck/eslint-config-holidaycheck/blob/master/baseConfig.json

I suppose the upside to that is that you can look at the configuration and actually see what the rules for the respective project are. Instead of writing some code to see if it lints. Of course that only applies if either the rule is obvious or you have some prior knowledge of the eslint rules.

I don't mind removing the disabled ones if you prefer that. TBH that is the result of that converting tool. I am not entirely sure why it picked the ones that are included and disabled. I think it does not make to explicitly disable *some* rules so I'll go ahead and just remove all disabled rules.